spec/fixtures/files/datahashes/core/exhibition1.json

```diff
@@ -12,7 +12,7 @@
     "organizerOrganizationUlan": "Signal",
     "venueOrganizationLocal": null,
     "venueOrganizationUlan": null,
-    "venueLocationLocal": "Khago",
+    "venueLocationLocal": "Khago;",
     "venueLocationOffsite": "Stay",
```

Comment:
With no delimiter after Khago, this creates a combined `venueLocation` field: `Khago;Stay`.

Since there is a row with blank `venueLocation` between Khago and Stay, I added a delimiter to the end, so I get the following, which lines up properly with the other fields: 

`Khago;;Stay`